### PR TITLE
Deploy latest dashboard to robocat

### PR DESCRIPTION
**Pull request by afrittoli**
# Changes

Add an overlay for dashboard and a cronjob to deploy dashboard nightly readonly builds to robocat.

Work on #224
Depens on #272 #275 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: tekton/cronjobs/robocat-certificates-on-demand/cronjob.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: folder-cd-trigger
+spec:
+  suspend: true
+  schedule: "1 1 1 1 1"
```
Comment (bobcatfish) [inline tekton/cronjobs/robocat-certificates-on-demand/cronjob.yaml]: once a year?

File: tekton/cronjobs/robocat-certificates-on-demand/README.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Cron Job to daily apply the resources in the robocat/cadmin folder, from the
```
Comment (bobcatfish) [inline tekton/cronjobs/robocat-certificates-on-demand/README.md]: not sure if "daily" is accurate given the cron schedule? (i am far from a cron expert tho 😅)

File: tekton/cronjobs/robocat-certificates-on-demand/cronjob.yaml (hunk 1 above)
Comment (afrittoli) [reply tekton/cronjobs/robocat-certificates-on-demand/cronjob.yaml]: Heh, no, it's `suspend: true`, so it only runs on demand.
It must be run once after the cert-manager is deployed.
The second one in the table here: https://github.com/tektoncd/plumbing/blob/2c53795759c4dc5d2d928279da20cc002dc32fb0/robocat/README.md#run-the-cronjobs

File: tekton/cronjobs/robocat-certificates-on-demand/README.md (hunk 2 above)
Comment (afrittoli) [reply tekton/cronjobs/robocat-certificates-on-demand/README.md]: Upz, copy paste issue :P

**Pull request merged**